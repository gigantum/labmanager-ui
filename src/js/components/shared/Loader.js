//vendor
import React, { Component } from 'react'

export default class Loader extends Component {

  render() {
    return (
      <div className="Loader">
      </div>
    )
  }
}
