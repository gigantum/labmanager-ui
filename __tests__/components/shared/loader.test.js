import React, { Component } from 'react'
import renderer from 'react-test-renderer';
import Loader from 'Components/shared/Loader';

test('Test Loader rendering', () => {
  const component = renderer.create(

    <Loader/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
