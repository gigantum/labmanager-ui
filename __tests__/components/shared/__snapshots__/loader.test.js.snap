// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loader rendering 1`] = `
<div
  className="Loader"
/>
`;
